tracer/test/test-applications/integrations/Samples.CIVisibilityVersionMismatch/Samples.CIVisibilityVersionMismatch.csproj

```diff
@@ -0,0 +1,37 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <IsPackable>false</IsPackable>
+    <OutputType>Library</OutputType>
+
+    <!-- Required to build multiple projects with the same Configuration|Platform -->
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <GenerateTargetFrameworkAttribute>false</GenerateTargetFrameworkAttribute>
+    <GenerateRuntimeConfigurationFiles>true</GenerateRuntimeConfigurationFiles>
+    <GenerateProgramFile>false</GenerateProgramFile>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.7.1" />
```

Comment:
Do you think we should be testing this against multiple versions of xunit (and hence multiple versions of the test sdk?)

Reply:
I've tested in 16.7.1 and 17.6.0, I don't know if it worth testing in all versions, the SDK code for this class haven't changed since v15.5.0 (2017). They fixed the assembly version to 15 so they are forbidden to make major changes in the api.